pgxscan/pgxscan.go

```diff
@@ -55,12 +55,18 @@ func ScanAll(dst interface{}, rows pgx.Rows) error {
 // See dbscan.ScanOne for details.
 func ScanOne(dst interface{}, rows pgx.Rows) error {
```

Comment:
Done

Reply:
> If no rows are found we return a `pgx.ErrNoRows` error, otherwise nil.

I think this statement is wrong because otherwise, we propagate the error from dbscan, it might be non-nil.
The better is just: 
`If no rows are found it returns a pgx.ErrNoRows error`

Reply:
Updated